app/src/main/java/com/bongtu/baekseo/presentation/onboarding/component/OnBoardingBottomSheet.kt

```diff
@@ -0,0 +1,291 @@
+            .background(color = BongBaekTheme.colors.gray750)
+            .padding(horizontal = 20.dp),
+    ) {
+        Text(
+            text = stringResource(id = onboarding_bottom_sheet_description),
+            modifier = Modifier.padding(top = 40.dp),
+            style = BongBaekTheme.typography.titleSemiBold18,
+            color = BongBaekTheme.colors.white,
+        )
+
+        Spacer(modifier = Modifier.size(30.dp))
+
+        OnBoardingAgreeItem(
+            item = OnBoardingAgree(
+                titleRes = onboarding_bottom_sheet_check_all,
+                isDescription = true,
+                isArrowVisible = false,
+                isChecked = allChecked,
+            ),
+            onCheckedChange = onAllCheckedChange,
+        )
+
+        HorizontalDivider(
+            modifier = Modifier
+                .fillMaxWidth()
+                .width(1.dp)
+                .padding(vertical = 18.dp),
+            color = BongBaekTheme.colors.lineNormal,
+        )
+
+        items.forEachIndexed { index, item ->
+            OnBoardingAgreeItem(
+                item = item,
+                onCheckedChange = { checked ->
+                    onItemCheckedChange(index, checked)
+                },
+            )
+            if (index < items.lastIndex) {
+                Spacer(modifier = Modifier.size(14.dp))
+            }
+        }
+
+        BongBaekButton(
+            title = "다음",
+            onClick = onNextClick,
+            buttonType = ButtonType.PRIMARY,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 40.dp),
+        )
+    }
+}
+
+@Composable
+private fun OnBoardingAgreeItem(
+    item: OnBoardingAgree,
+    onCheckedChange: (Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+    onIconClick: () -> Unit = {},
+) {
+    val (textStyle, textColor) = if (item.isDescription) {
+        BongBaekTheme.typography.titleSemiBold16 to BongBaekTheme.colors.gray200
+    } else {
+        BongBaekTheme.typography.body2Regular16 to BongBaekTheme.colors.gray300
+    }
+
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .wrapContentHeight(),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        BongBaekCheckBox(
+            isChecked = item.isChecked,
+            onCheckedChange = onCheckedChange,
+            checkBoxType = CheckBoxType.PRIMARY,
+        )
+
+        Text(
+            text = stringResource(id = item.titleRes),
+            modifier = Modifier.padding(start = 8.dp),
+            style = textStyle,
+            color = textColor,
+        )
+
+        Spacer(modifier = Modifier.weight(1f))
+
+        if (item.isArrowVisible) {
+            Icon(
+                imageVector = ImageVector.vectorResource(id = ic_arrow_right),
+                contentDescription = null,
+                modifier = Modifier.noRippleClickable {
+                    onIconClick()
+                },
```

Comment:
p2: 아래처럼 함수 전달하면 더 좋을거 같아요 !
```suggestion
                modifier = Modifier.noRippleClickable(onClick = onIconClick),
```